x-pack/plugins/cloud_security_posture/public/pages/findings/findings.test.tsx

```diff
@@ -44,12 +44,22 @@ describe('<Findings />', () => {
   it("renders the success state component when 'kubebeat' DataView exists and request status is 'success'", async () => {
     const data = dataPluginMock.createStartContract();
     const unifiedSearch = unifiedSearchPluginMock.createStartContract();
-    const source = await data.search.searchSource.create();
 
     (useCisKubernetesIntegration as jest.Mock).mockImplementation(() => ({
       data: { item: { status: 'installed' } },
     }));
-    (source.fetch$ as jest.Mock).mockReturnValue(of({ rawResponse: { hits: { hits: [] } } }));
+
+    (data.search.search as jest.Mock).mockReturnValue(
+      of({
+        rawResponse: {
+          aggregations: {
+            groupBy: { buckets: [] },
+            count: { buckets: [] },
+          },
+          hits: { hits: [] },
+        },
+      })
+    );
```

Comment:
this test changed because we replaced the `searchSource` with directly calling `search`, so this change reflects that. 

side-note: the mocked response is kinda fake, as it includes `groupBy` and `count` combined, although those are actually return in different responses. i didn't think this mattered in this test, so didn't put the effort to account for that. if you feel it's needed, LMK